### PR TITLE
Update tutorials for beta2

### PR DESCRIPTION
**Pull request by maier49**
This includes:
* Fixes for grammatical errors or typos
* Fixes for anything broken in the demo code right now
* Changes needed to work with the latest (as of now)  versions of Dojo 2 dependencies.

```
@dojo/cli-build-webpack@2.0.0-beta2.3
@dojo/core@2.0.0-beta2.3
@dojo/has@2.0.0-beta2.2
@dojo/i18n@2.0.0-beta2.2
@dojo/shim@2.0.0-beta2.3
@dojo/cli-test-intern@2.0.0-beta2.2
@dojo/interfaces@2.0.0-beta2.2
@dojo/loader@2.0.0-beta2.2
@dojo/routing@2.0.0-beta2.3
@dojo/widget-core@2.0.0-beta2.5
@dojo/widgets@2.0.0-alpha.24
@dojo/test-extras@2.0.0-alpha.4
```

### CODE REVIEW

File: site/source/tutorials/005_form_widgets/index.md [hunk 1]
```diff
@@ -149,47 +149,33 @@ The form is now ready to be integrated into the application. We will do that in
...
+Now that the `WorkerForm` widget is complete, we will update the `App` class to use it. First, we need to address how to store the user-completed form data. Recall that the `WorkerForm` will accept an `onFormInput` property that will allow the `App` class to be informed when a field value changes. However, the `App` class does not currently have a place to store those changes. We'll add a private property to the `App` to store this state, and a method to update the state and re-render the parts of the application that have changed. As the application grows and needs to store more data, using private properties on a widget class can become difficult to maintain. Dojo 2 uses containers and injectors to help manage the complexities of dealing with state in a large application. For more information, refer to the [Containers and Injecting State](../comingsoon.html) article.
```
Comment (dylans) [inline site/source/tutorials/005_form_widgets/index.md]: We'll -> We will (the war on contractions other than let's) :)

File: site/source/tutorials/002_creating_an_application/demo/finished/biz-e-corp/package.json [hunk 2]
```diff
@@ -2,19 +2,20 @@
...
   "devDependencies": {
-    "@dojo/cli-build-webpack": "beta1",
-    "@dojo/cli-test-intern": "beta1",
-    "@dojo/interfaces": "beta1",
-    "@dojo/loader": "beta1",
+    "@dojo/cli-build-webpack": "beta2",
+    "@dojo/cli-test-intern": "beta2",
+    "@dojo/interfaces": "beta2",
+    "@dojo/loader": "beta2",
+    "@dojo/test-extras": "2.0.0-beta2.1",
```
Comment (kitsonk) [inline site/source/tutorials/002_creating_an_application/demo/finished/biz-e-corp/package.json]: We will have a `beta2` tag for `test-extras`

**Comment by evra** [comment]
Could you please merge this commit? The current version of the tutorial is incompatible with beta 2 and thus misleading. Thank you.

**Comment by kitsonk** [comment]
@evra while our packages are at `next` are what will likely become beta2, we haven't released and tagged `beta2` yet and we weren't going to release this PR until likely a couple days after we did the release.  The `beta2` release is likely to be this week and then we need to rebase this PR and double check any other changes.

**Comment by maier49** [comment]
Closing in favor of #165 which has been rebased and includes additional updates.

**Pull request closed without merging**